docs/userguides/basics.md

```diff
@@ -37,6 +37,23 @@ def promptForPassword():
 sdk = py42.sdk.from_local_account("https://console.us.code42.com", "my_username", "my_password", totp=promptForPassword)
 ```
 
+Alternatively, define a function that returns the auth token based on user's authentication approach
+
+```
+def jwt_provider():
+    import json
+    import requests
+    from requests.auth import HTTPBasicAuth
```

Comment:
is it fine to have the import statements inside the function? or since its just an example may be we can skip those statements!!

Reply:
You could put it above the function

Reply:
Done.